ventilator_alarm.py

```diff
@@ -46,6 +46,11 @@ def run(self, name):
                     self.time_last_kick_received == cur_time
                     if msg['val'] != 0:
                         self.request_queue.put({'type': 'error', 'value': msg['val']})
+                elif msg['type'] == "PRES":
+                    if ((msg['val'] > proto.settings_values['PK'] + proto.settings_values['ADPK']) or
+                       (msg['val'] < proto.settings_values['PK'] - proto.settings_values['ADPK'])):
+                        self.request_queue.put({'type': 'alarm', 'priority': 42, 'value': 5})
```

Comment:
5 seems a strange value if we're still using bit field alarms. Ideally we'd have a list of the definitions available somewhere. Is this a placeholder?